ts/components/ui/Markdown/index.tsx

```diff
@@ -48,12 +48,56 @@ function renderMarkdown(body: string): React.ReactNode {
   }
 }
 
+interface Props {
+  children: string;
+  lazy?: boolean;
+}
+
+interface State {
+  renderedMarkdown: ReturnType<typeof renderMarkdown> | undefined;
+}
+
 /**
  * A component that accepts "markdown" as child and render react native
  * components.
  */
-const Markdown: React.SFC<{
-  children: string;
-}> = props => <View>{renderMarkdown(props.children)}</View>;
+class Markdown extends React.PureComponent<Props, State> {
+  public static defaultProps: Partial<Props> = {
+    lazy: false
+  };
+
+  constructor(props: Props) {
+    super(props);
+
+    this.state = {
+      renderedMarkdown: undefined
+    };
+  }
+
+  public componentDidMount() {
+    if (this.props.lazy) {
+      // Render the markdown string asynchronously.
+      setTimeout(
```

Comment:
It's [recommended](https://facebook.github.io/react-native/docs/timers#interactionmanager) to use `InteractionManager.runAfterInteractions` to avoid blocking animations, see: https://github.com/teamdigitale/italia-app/blob/master/ts/components/screens/MarkdownScreenComponent.tsx#L36

Also please update `MarkdownScreenComponent` and the `ContextualHelp` component